Test-№1.1-from-19.10.2022/task3.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stddef.h>
+#include <string.h>
+#define maxSize 500
+
+int countOfSymbols(char *path)
+{
+    FILE *testFile = fopen(path, "r");
+    char symbol = "c";

```

Comment:
```suggestion
    char currentSymbol = '\0';

```
Инициализация символа строкой?
И имя переменное - так себе.